tools/hotspots_annotation.cwl

```diff
@@ -0,0 +1,29 @@
+cwlVersion: v1.0
+class: CommandLineTool
+id: hotspots_annotation
+requirements:
+  - class: ShellCommandRequirement
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    ramMin: ${ return inputs.ram * 1000 }
+    coresMin: $(inputs.cores)
+  - class: DockerRequirement
+    dockerPull: 'dmiller15/hotspots:0.1.0'
+
+baseCommand: []
+
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      $(inputs.disable_hotspot_annotation ? ">&2 echo 'User elected to skip hotspot annotation' && exit 0;" : ">&2 /hotspot.py")
```

Comment:
What does the `>&2` do when putting it before the commands?

Reply:
The same thing as putting it after the command. I think putting it after is more common. I can put it at the end to reduce confusion.

Reply:
If you're asking in general, it allows error messages that would go to stdout to stderr so that `job.err.log` in cavatica will catch it

Reply:
Cool.

Reply:
You know what, neither did I!